mlrun/projects/project.py

```diff
@@ -2335,12 +2336,13 @@ def build_function(
         :param base_image:      base image name/path (commands and source code will be added to it)
         :param commands:        list of docker build (RUN) commands e.g. ['pip install pandas']
         :param secret_name:     k8s secret for accessing the docker registry
-        :param requirements:    list of python packages or pip requirements file path, defaults to None
+        :param requirements:    list of python packages, defaults to None
         :param mlrun_version_specifier:  which mlrun package version to include (if not current)
         :param builder_env:     Kaniko builder pod env vars dict (for config/credentials)
                                 e.g. builder_env={"GIT_TOKEN": token}, does not work yet in KFP
         :param overwrite_build_params:  overwrite the function build parameters with the provided ones, or attempt to
          add to existing parameters
+        :param requirements_file:    pip requirements file path, defaults to None
```

Comment:
My opinion is that the docstring should be organized logically (even if the parameter order is forced differently), so I would put the docstring of this parameter after the one for `requirements`.

Reply:
Won't it be confusing when using it without kwargs?
Or will the hinting show the correct order anyways?

Reply:
I organized it in a more logical way 👍